integrations/client/test_delphi_epidata.py

```diff
@@ -17,6 +17,14 @@
 from delphi_utils import Nans
 
 
+# TODO replace these real geo_values with fake values, and use patch and mock to mock the return values of 
+# delphi_utils.geomap.GeoMapper().get_geo_values(geo_type) in parse_geo_sets() of _params.py
+
+global fips, msa # add two global lists of valid geo_values of these types to pass the validation in parse_geo_sets() of _params.py
+
+fips = ['04019', '19143', '29063'] # Example list of valid FIPS codes as strings
+msa = ['40660', '44180', '48620'] # Example list of valid MSAs as strings
+
```

Comment:
you might wanna put these into [src/acquisition/covidcast/test_utils.py](https://github.com/cmu-delphi/delphi-epidata/blob/a6cc05838a19aa5af977df29c360b8bb5a95054a/src/acquisition/covidcast/test_utils.py) so they can be shared with other tests and dont have to be repeatedly defined in each test file.  you could maybe even make them defaults in `CovidcastTestRow` in `test_utils.py`, and you wont have to specify them in every call to `make_default_row()`!

Reply:
@melange396 thanks for jumping in on this review! i should provide some context for this particular change though: initially the plan was to mock out the GeoMapper calls in tests so that we wouldn't need to use actual FIPS and MSA codes. Leon hasn't had much mocking experience though, and we tried multiple ways to get it to work with no luck. so having these constants is a sortof stopgap to get this PR out, but i don't think they're a good solution long term. 

what do you think, keep the constants, or find a mock wizard (a wizard at mock, not a sham magician) to take a look?

Reply:
i dont think using those constants is all that _**bad**_, though not as nice as a mock.  one argument against using mocks is that theyll hide (from the tests, at least) any changes to the arguments or output of the GeoMapper.  i think the constants can stick around for now (especially if we put them in one place), but definitely keep that TODO comment along with them.